programs/ssl/ssl_base64_dump.c

```diff
@@ -0,0 +1,953 @@
+/*
+ *  SSL deserializer from base64 code
+ *
+ *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
```

Comment:
Please update this header with a good date.

Reply:
All programs in this folder have the same copyright date. It will change it of course, but please tell me what date should be placed here